src/components/App.tsx

```diff
@@ -2377,17 +2378,25 @@ class App extends React.Component<AppProps, AppState> {
           element,
           this.state,
           [scenePointer.x, scenePointer.y],
-          this.isMobile,
+          this.isMobile || isIPad,
         ) &&
         index <= hitElementIndex
       );
     });
   };
 
   private redirectToLink = () => {
+    const draggedDistance = distance2d(
+      this.lastPointerDown!.clientX,
+      this.lastPointerDown!.clientY,
+      this.lastPointerUp!.clientX,
+      this.lastPointerUp!.clientY,
+    );
     if (
-      this.lastPointerDown!.clientX !== this.lastPointerUp!.clientX ||
-      this.lastPointerDown!.clientY !== this.lastPointerUp!.clientY
+      // strict check if not on iPad or Mobile to ensure pointerdown
+      //  and pointerup is same point
+      (!(this.isMobile || isIPad) && draggedDistance !== 0) ||
```

Comment:
I hope I am reading it right, won't this mean that for desktop devices when dragging distance is not zero, it will allow threshold?
To make it clear
lets change the assertion to positive -> check for threshold only for mobile / ipad